public/pages/Alerts/components/AlertFlyout/AlertFlyout.tsx

```diff
@@ -113,12 +108,23 @@ export class AlertFlyout extends React.Component<AlertFlyoutProps, AlertFlyoutSt
       });
 
       if (ruleIds.length > 0) {
-        const rulesResponse = await this.rulesViewModelActor.fetchRules({
-          _id: ruleIds,
+        const rules = await DataStore.rules.getAllRules({
+          from: 0,
+          size: 5000,
+          query: {
+            nested: {
+              path: 'rule',
+              query: {
+                terms: {
+                  _id: ruleIds,
+                },
+              },
+            },
+          },
```

Comment:
Shall we only pass the terms object `{ _id: ruleIds }` and have the rest as placeholder in the RulesStore?

Reply:
I see we use same options with different terms in the CreateDetector.tsx as well

Reply:
Let me check.

Reply:
code base updated.